e2e/k8s_test.go

```diff
@@ -132,3 +152,238 @@ func TestLargeDecodedVariableSecretProvidedK8s(t *testing.T) {
 
 	testenv.Test(t, f.Feature())
 }
+
+func TestMultiplePodsChangingPwdInbetweenSecretProvidedK8s(t *testing.T) {
```

Comment:
I've replicated the following flow for this particular test based off of `TEST_ID_2_multiple_pods_changing_pwd_inbetween`:

1. first, check original `TEST_SECRET` value and confirm it's `supersecret`
2. determine which platform we're running on and set Conjur secret (`secret2`) for pod2
3. check `TEST_SECRET` value and confirm it's `secret2`
4. set Conjur secret (`secret3`) for pod3
5. check `TEST_SECRET` value and confirm it's `secret3`

I have not (and currently unsure how to) implemented the `cli_with_timeout` logic as well as waiting for `x` amount of pods running, `scale deployment`, etc.

Further feedback is appreciated!

Reply:
I would recommend looking into the e2e-framework and Kubernetes Go APIs a little bit. I would be surprised if there is not a way to scale deployments/delete pods through the k8s client

If not, then it may be necessary to add a helper script which executes the kubectl/oc commands you need